controller/src/main/java/io/pravega/controller/server/eventProcessor/LocalController.java

```diff
@@ -86,28 +87,28 @@ public LocalController(ControllerService controller, boolean authorizationEnable
 
     @Override
     public CompletableFuture<Boolean> checkScopeExists(String scopeName) {
-        return Futures.exceptionallyExpecting(this.controller.getScope(scopeName).thenApply(v -> true),
+        return Futures.exceptionallyExpecting(this.controller.getScope(scopeName, 0L).thenApply(v -> true),
```

Comment:
Are these `0L` the request IDs? Because the `RequestTracker` identifies requests based on the ID + request type. So, 2 successive requests of any of these methods would be undistinguishable from one another as the ID is the same.

Reply:
Yes, these 0L are request ids. 
these apis bypass grpc, hence request tracker doesnt track them. 
The local controller in controller service is used to basically bypass the call to controller from pravega clients used in controller. 
These clients are used to write events into internal streams or read them back.

so we dont really care to track these requests per se hence i had used 0L. but yes, using `0L` is quite pointless. I am changing it to generate a new request id in such situations.